src/model/output.ts

```diff
@@ -8,6 +8,10 @@ class Output {
   static async setAndroidVersionCode(androidVersionCode: string) {
     core.setOutput('androidVersionCode', androidVersionCode);
   }
+
+  static async setExitCode(exitCode: number) {
+    core.setOutput('exitCode', exitCode);
```

Comment:
Do you think it will be helpful to reflect that this exit code comes from the editor?

Im thinking that the exit code might be different from the CLI exit code under some circumstances.

Perhaps editorExitCode?